src/local/index.js

```diff
@@ -80,15 +80,17 @@ function startProject(config, projectConf) {
 
     // WS events
   wsClient.on(wsEvents.READY, () => {
-    triggerBigSync(projectConf, _.pick(config, ['debug', 'delete']), () => {
-      fsHelper.watch();
-
-      localLog(
-        text.SYNC_ON_CONNECT,
-        projectConf.hostname, (projectConf.prefersEncrypted) ? 'using' : 'not using',
-        'encryption'
-      );
-    });
+    if (projectConf.triggerBigSync !== false) {
```

Comment:
Can probably just do `if (projectConf.triggerBigSync) {`. Older configurations won't have this definition, and will return`undefined`, which is falsy.